src/plugins/recordTypes/index.js

```diff
@@ -25,9 +25,10 @@ import objectexit from './objectexit';
 import organization from './organization';
 import person from './person';
 import place from './place';
-// import pottag from './pottag';
+import pottag from './pottag';
```

Comment:
Comment this out until it is moved, so it doesn't needlessly increase the size of the JS bundle.